moveit_fake_controller_manager/src/moveit_fake_controller_manager.cpp

```diff
@@ -324,15 +260,19 @@ class MoveItFakeControllerManager : public moveit_controller_manager::MoveItCont
   }
 
   /* Cannot switch our controllers */
-  virtual bool switchControllers(const std::vector<std::string> &activate, const std::vector<std::string> &deactivate) { return false; }
+  virtual bool switchControllers(const std::vector<std::string> &activate, const std::vector<std::string> &deactivate)
+  {
+    return false;
+  }
 
 protected:
-
   ros::NodeHandle node_handle_;
-  std::map<std::string, moveit_controller_manager::MoveItControllerHandlePtr> controllers_;
+  ros::Publisher pub_;
+  std::map<std::string, BaseFakeControllerPtr> controllers_;
 };
 
-} // end namespace moveit_fake_controller_manager
+}  // end namespace moveit_fake_controller_manager
 
+#include <pluginlib/class_list_macros.h>
```

Comment:
I think you could make that argument for all sorts of things that are included. still, elsewhere in moveit and most all other code i've ever seen its always at the top of the file